pandas/core/frame.py

```diff
@@ -2939,7 +2935,11 @@ def to_html(
             render_links=render_links,
         )
 
-    @doc(storage_options=generic._shared_docs["storage_options"])
+    @doc(
```

Comment:
i am puzzled how this works here. shouldn't this be just shared_docs? (and not generic)

Reply:
I think it’s aliases because it’s imported there. Will change any generic to shared directly